### PR TITLE
Adding Python Script for Rock Paper Scissor Game

### PR DESCRIPTION
**Pull request by gulshanbaraik01**
Hello,

This is my PR #4 for Awesome Python Scripts. The second one for Basic Scripts.
By this script, we can play a simple game of Rock Paper Scissor. Please review and add it.

Thanks,
Gulshan Baraik

### CODE REVIEW

File: Basic-Scripts/Rock_paper_scissor.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+player_two={0:'Paper',1:'Rock',2:'Scissor'}
+while(1):
+    num1=input("Player one, Enter your choice")
+    num2=input("Player two, Enter your choice")
+    bit1=int(input("Player one, Enter the secret bit position"))
+    bit2=int(input("Player two, Enter the secret bit position"))
+    rock_paper_scissor(num1,num2,bit1,bit2)
+    ch=input("Do you want to continue? y/n")
+    if(ch=='n'):
+        break
```
Comment (SKAUL05) [inline Basic-Scripts/Rock_paper_scissor.py]: Include a main function and document your code properly.
Follow PEP8 Guidelines for naming conventions and indentations.
https://www.python.org/dev/peps/pep-0008/

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/Rock_paper_scissor.py (hunk 1 above)
Comment (gulshanbaraik01) [reply Basic-Scripts/Rock_paper_scissor.py]: Hello,

I updated as per your feedback. Please review and add it.

File: Basic-Scripts/Rock_paper_scissor.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug 17 14:24:23 2020
+
+@author: Gulshan
+"""
+# Function rock_paper_scissor--->
+def rock_paper_scissor(num1,num2,bit1,bit2):
+    p1=int(num1[bit1])%3
```
Comment (SKAUL05) [inline Basic-Scripts/Rock_paper_scissor.py]: @gulshanbaraik01 Use meaningful and readable names instead of p1 p2

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/Rock_paper_scissor.py (hunk 2 above)
Comment (gulshanbaraik01) [reply Basic-Scripts/Rock_paper_scissor.py]: Hey, I updated it as per your recent feedback. Kindly check and add it.

**Comment by gulshanbaraik01** [comment]
Hey, I updated it 6 days ago as per your recent feedback. Kindly check and add it.

**Review by SKAUL05 — APPROVED** [review]
Good to go:)
Review and merge @ankitdobhal

**Pull request merged**